### ISSUE TITLE
Pixel icones

### ISSUE THREAD

**Issue by tobyplowy** [DreamBlocks/dreamblocks.github.io#11]
@Gerold55 this maybe Overkill but I think it would look good if the Social Network icones wuld be pixelart

**Comment by tobyplowy** [comment]
like this 

facebook
![fb](https://user-images.githubusercontent.com/16853304/27516938-8af27e30-59c3-11e7-89ec-d637093479ca.png)
twitter
![twit](https://user-images.githubusercontent.com/16853304/27516958-dbfb432a-59c3-11e7-9758-3e3c101920ed.png)
youtube
![youtube](https://user-images.githubusercontent.com/16853304/27516940-8b14a88e-59c3-11e7-82e7-c1f081d31060.png)
github
![github](https://user-images.githubusercontent.com/16853304/27516947-ae1f36fa-59c3-11e7-8a92-b99797e22998.png)

**Comment by LordStephen77** [comment]
@tobyplowy Yes. I used these because they are recallable from google's online library.

You can easily render them pixel style, using the noise filter on photoshop or gimp.

**Comment by tobyplowy** [comment]
@LordStephen77 no those images above are already pixelart (do you like them? i made them)

**Comment by LordStephen77** [comment]
@tobyplowy Yeah, i did not notice the pixel effect, my notebook has 15 inch screen. The pictures are small, but they are perfect! I can scaling through code, no problem, and the effect is even more beautiful.

Tomorrow I implement a beautiful thing on the website! A fantastic thing that will make the official website much more beautiful.

Thanks toby ;)

**Comment by Gerold55** [comment]
Is it possible to add the new YT logo with the old one?

**Comment by tobyplowy** [comment]
@Gerold55 I Don't really understand what you are saying what I understand is 

You whant me to make a new yt logo in the style of the old yt logo but in pixel art correct?

**Comment by Gerold55** [comment]
I was saying to add the play logo beside the old logo if that makes sense

**Comment by LordStephen77** [comment]
@Gerold55 You can do it all! The site I've created is easy to edit, and I'm working on my own custom css3 framework to add better animations, buttons and aesthetic content.

I need to create a logo for the project.

**Comment by tobyplowy** [comment]
@LordStephen77 shuld i also make icones for difrent platforms like windows android apple?

**Comment by LordStephen77** [comment]
@tobyplowy Android in the future, will only be possible with the use of Java 9.

IOS - I do not think I'll be making a version for iOS! In the future if new collaborators want to make an iOS ok version. I do not develop anything on Apple systems.

Windows, Mac OS, Linux Icons I've already got ready for you! That for Android will serve in the future. Is soon to announce a mobile version. The project will undergo many changes and improvements.

They are working on the buttons of the main game menu.
